src/UI/Pages/Resources/Summary.tsx

```diff
@@ -16,7 +19,19 @@ export const Summary: React.FC<Props> = ({ data }) =>
       loading: () => null,
       failed: () => null,
       success: (result) => (
-        <DeployStateChart summary={result.metadata.deploy_summary} />
+        <Toolbar>
```

Comment:
In the UI there is a lot of padding. This is because we have nested toolbars.
If you replace the `Toolbar` and `ToolbarContent` here with fragments, it is fixed.

Reply:
Ok, that's a good remark